internal/testutil/all_validators_test.go

```diff
@@ -0,0 +1,47 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type Validator interface {
+	Name() string
+	Run(utils.MetaBundle) (bool, error)
+	SucceedingCandidates() []utils.MetaBundle
+	FailingCandidates() []utils.MetaBundle
+}
+
+// register the validators to test here by appending them to the following slice
+var (
```

Comment:
nit: do we need the extra `( )` ?

Reply:
ahh, old habits, die hard! Would clean this up :)